examples/README.md

```diff
@@ -198,6 +198,61 @@ Update [create-nft-primary-txn.ts](./create-nft-primary-txn.ts#L10) with your ex
 yarn create-nft-primary-txn
 ```
 
+## Create Metadata Refresh
```

Comment:
Can we use sentence case for these headings?

Example:
```suggestion
## Create metadata refresh
```

Reply:
added sentence casing for the readme